docs/UserGuide.adoc

```diff
@@ -1,4 +1,4 @@
-= AddressBook Level 4 - User Guide
+= H.M.U v1.1 - User Guide
```

Comment:
User guide:
Should include v2.0 features, not just current features
Un-implemented features should be marked ‘coming in v2.0’ or something like that.